src/tss2-sys/sysapi_util.h

```diff
@@ -64,8 +64,6 @@ typedef struct {
     size_t nextData;
 } _TSS2_SYS_CONTEXT_BLOB;
 
-struct TSS2_SYS_CONTEXT;
```

Comment:
This looks like a random change that sould be its own commit (sorry for the pickiness)

Reply:
See #1244